_data/navigation.yml

```diff
@@ -424,7 +424,8 @@ primary:
             children: 
               - text: Lifecycle of a Launch 
                 url: lifecycle/
-
+              - text: Infrastructure
+                url: /launching-software/infrastructure/
```

Comment:
@dylanirlbeck It looks like the other child urls do not contain a preceding slash.